### PR TITLE
Dbiface skeleton

### PR DESCRIPTION
**Pull request by jcr7467**
added functions to fetch and upload high scores

### CODE REVIEW

**Comment by jcr7467** [comment]
if this is not how you envisioned dbiface.py and something is wrong, im sorry i tried my best

File: client/DatabaseIface.py [hunk 1]
```diff
@@ -1,13 +1,44 @@
...
+    def get_highscores(self, n_scores, mode, score_kind):
+        #n_scores, number of scores to return
+        #mode, 'multiplayer' or 'singleplayer'
+        #score_kind, 'lifetime' or 'singlegame'
 
-    def add_highscore(self, new_score):
-        raise NotImplementedException
\ No newline at end of file
+        # let me answer that for you, no it has to be nScores and not n_scores
+        # bad practice to send underscores through http requests,
```
Comment (randallwc) [inline client/DatabaseIface.py:22]: that's only for SEO right lol

Comment (randallwc) [reply client/DatabaseIface.py:22]: https://stackoverflow.com/a/18450653/17356304

Comment (randallwc) [reply client/DatabaseIface.py:22]: all i care about is consistency so do whatever but do it always

**Comment by randallwc** [comment]
looks good but need to fix conflict

**Comment by randallwc** [comment]
https://github.com/ECE-180D-WS-2022/Team3/pull/155#issuecomment-1107442205

lol bruh what

i just want something that is useable

**Pull request merged**